examples/linux/io_epoll_test.cpp

```diff
@@ -51,6 +56,18 @@ using namespace unifex;
 using namespace unifex::linuxos;
 using namespace std::chrono_literals;
 
+#if !UNIFEX_NO_COROUTINES
+namespace unifex
+{
+  // get_exception_ptr customization point for std::error_code
+  std::exception_ptr
+  tag_invoke(tag_t<unifex::get_exception_ptr>, std::error_code&& ec) noexcept {
+    return std::make_exception_ptr(
+        std::system_error{std::forward<std::error_code>(ec)});
+  }
```

Comment:
This doesn't really belong here. Personally, I would bake it into the `get_exception_ptr` CPO itself since this is likely to be the behavior everybody is going to want to `std::error_code`.